### DISCUSSION TITLE
Which keepalived image is used?

### DISCUSSION THREAD

**Discussion by zhboseu** [mehdy/keepalived-exporter#143] [Q&A] [answered]
Hi
Which keepalived image you used?

<img width="866" alt="image" src="https://github.com/mehdy/keepalived-exporter/assets/17448274/0d8e32b8-e3e4-4624-9b60-78adadf72f02">

Thank you for your reply

**Comment by clwluvw** [comment, marked as answer]
You can use any image that the Keepalived process runs with PID 1. A sample would be something like this: https://github.com/mehdy/keepalived-exporter/blob/master/deployments/dev/keepalived.Dockerfile
